src/main/java/org/broadinstitute/hellbender/tools/copynumber/coverage/segmentation/CopyRatioSegmentCollection.java

```diff
@@ -0,0 +1,49 @@
+package org.broadinstitute.hellbender.tools.copynumber.coverage.segmentation;
+
+import org.broadinstitute.hellbender.tools.copynumber.formats.collections.SampleLocatableCollection;
+import org.broadinstitute.hellbender.tools.copynumber.formats.metadata.SampleMetadata;
+import org.broadinstitute.hellbender.utils.SimpleInterval;
+import org.broadinstitute.hellbender.utils.tsv.DataLine;
+import org.broadinstitute.hellbender.utils.tsv.TableColumnCollection;
+
+import java.io.File;
+import java.util.List;
+import java.util.function.BiConsumer;
+import java.util.function.Function;
+
+public final class CopyRatioSegmentCollection extends SampleLocatableCollection<CopyRatioSegment> {
+    enum CopyRatioSegmentTableColumn {
+        CONTIG,
+        START,
+        END,
+        NUM_POINTS_COPY_RATIO,
```

Comment:
If this is a copy ratio segment, why does `COPY_RATIO` need to be made explicit in the variable name?

Reply:
This is to distinguish this column from NUM_POINTS_ALLELE_FRACTION in tables where both are present.  Although we could go with NUM_POINTS for both copy-ratio-only/allele-fraction-only tables, I think there's no real disadvantage to being a little more explicit.